requirements/doc.txt

```diff
@@ -5,21 +5,27 @@
 #    pip-compile --output-file requirements/doc.txt requirements/base.in requirements/doc.in
 #
 alabaster==0.7.9          # via sphinx
+amqp==2.1.4               # via kombu
```

Comment:
I'm not sure.  Perhaps sphinx needs the be able to import the app to build its docs?  The relationship between base.in and doc.txt is defined by the cookiecutter, so @jmbowman may be better equipped to answer the question.